package.json

```diff
@@ -34,8 +37,10 @@
     "@vue/eslint-config-prettier": "^5.0.0",
     "@vue/eslint-config-typescript": "^4.0.0",
     "babel-loader": "^8.0.6",
+    "cypress": "^4.0.2",
     "eslint": "^5.16.0",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-cypress": "^2.10.3",
+    "eslint-plugin-prettier": "^3.1.1",
```

Comment:
Do we need to support an older version of prettier? Or was this automatically changed by cypress?